packages/cli/src/routeGeneration/templates/express.hbs

```diff
@@ -19,10 +19,9 @@ const promiseAny = require('promise.any');
 import { iocContainer } from '{{iocModule}}';
 import { IocContainer, IocContainerFactory } from '@tsoa/runtime';
 {{/if}}
-import type { RequestHandler } from 'express';
```

Comment:
minor unrelated suggestion, only importing the type makes the intent of the file clearer